alma_tests_cacher/cacher.py

```diff
@@ -177,7 +177,12 @@ async def process_repo(
             repo_dir = Path(workdir, repo_dirname)
             if not repo_dir.exists():
                 self.logger.info('Start cloning git repo: %s', repo.url)
-                exit_code, stdout, stderr = clone_git_repo(workdir, repo.url)
+                try:
+                    exit_code, stdout, stderr = clone_git_repo(workdir, repo.url)
+                except Exception as e:
+                    message = f"Cannot clone git repo: {str(e)}"
+                    self.logger.exception('Cannot clone git repo: %s', str(e))
```

Comment:
You already have the message defined above, why not using it?

Reply:
I initially wanted to do this but I've seen that logging is made in such way several times in `albs-web-server` and thought it is made on purpose, so that's why I didn't use